docs/platform/webcontainers/browser-support.md

```diff
@@ -12,25 +12,40 @@ head:
 
 _Last update: February 2023_
 
-**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in beta. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
+## Supported browsers
 
-:::warning Note
-There is a reported Chrome regression on Macbooks with M1 chip, which also affects the speed of some larger projects on WebContainers. Learn more about this issue in these bug reports: [issue 1228686](https://bugs.chromium.org/p/chromium/issues/detail?id=1228686) and [issue 1356099](https://bugs.chromium.org/p/chromium/issues/detail?id=1356099).
-:::
+WebContainers are supported in all recent desktop browsers (fully in Chrome and Chrome-based browsers, in beta in Firefox and Safari), and partially supported on Android browsers.
+
+### Desktop browsers
+
+- Chrome: full support.
+- Other Chromium-based browsers: full support. Some browsers may apply restrictive rules by default, and require specific configuration, like [Brave](#brave).
+- Firefox: beta support ([see details](#firefox)).
+- Safari: beta support since Safari 16.4, no support for earlier versions ([see details](#safari)).
+
+### Mobile browsers
+
+- Chrome, Chromium-based browsers and Firefox on Android: beta support. Depending on your device, large projects may run into memory limitations.
+- Safari on iOS 16.4: experimental support. Many projects may run into memory limitations and other compatibility issues.
```

Comment:
Not yet supported though. I mean the editor will launch but WebContainer won't boot until we have a separate build for iOS and iPadOS. Maybe it also makes sense to separate iOS and iPadOS as there is overlap but the memory restrictions are different between both and tablets have a little higher memory limits. iOS will be a lot more restrictive and I don't think iOS is something we should support because almost no project with dependencies will run, and it would also mean we'd need a build for iOS and another one for iPadOS. Plus I think we can't even distinguish between the two via the user agent, can we?

Reply:
we can distinguish via user agents as ipad gives a desktop UA. IOS gives a mobile UA. We can check for touch support to differentiate between desktop safari and ipad safari

Reply:
Yea we may need to be a little more clever if it's possible to detect. I haven't looked into it but if you say we can then that'd be good. But we do want to mention the differences in memory restrictions between iPad and iOS.